### PR TITLE
[CS] What to play

### PR DESCRIPTION
**Pull request by tomaas-zeman**
cs-04
en-04

### CODE REVIEW

File: sections/what_to_play.tex [hunk 1]
```diff
@@ -28,3 +28,5 @@ \subsection*{Recommendations}
 \end{multicols}
 
 \includegraphics[width=\linewidth]{\art/enchanter.jpg}
+
+\vspace*{-1em}
```
Comment (qwrtln) [inline sections/what_to_play.tex:32]: That moves the enchanter down?

**Comment by qwrtln** [comment]
Interesting that `po4a` is not complaining.

File: sections/what_to_play.tex (hunk 1 above)
Comment (tomaas-zeman) [reply sections/what_to_play.tex:32]: Yes. As you can see on the 'cs' comparison, the layout was not quite right, most likely does to tiny differences between spacings

Comment (qwrtln) [reply sections/what_to_play.tex:32]: OK, nicely done.

**Pull request merged**